gulpfile.babel.js

```diff
@@ -83,9 +85,43 @@ gulp.task('bootstrap_test', () => {
   gulp.src(['bootstrap/test/test-*.js'], {read: false})
     .pipe(mocha())
     .once('error', () => {
-	process.exit(1);
+      process.exit(1);
     })
     .once('end', () => {
-	process.exit();	
+      process.exit();
     })
-});
\ No newline at end of file
+});
+
+gulp.task('webpack', () => {
```

Comment:
Let's use |build_client| instead of |webpack|.
Also, existing |build| should be |build_server|.